core/embed/rust/src/ui/display/loader/circular_thin.rs

```diff
@@ -0,0 +1,335 @@
+use crate::ui::{
```

Comment:
aside: this whole file is a copy-paste job and probably shouldn't be. but we don't want to spend that time in this PR.

does the new drawing library do something about it? @cepetr

Reply:
@matejcik: with the new drawing library, there will be no more "circular_xxx.rs" files; these "loaders" are implemented as just a few lines of code directly in place